components/drafts/actions/legacy_drafts.ts

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {batchActions} from 'redux-batched-actions';
+
+import {getPost} from 'mattermost-redux/actions/posts';
+import {DispatchFunc, GenericAction, GetStateFunc} from 'mattermost-redux/types/actions';
+
+import {getAllPosts} from 'mattermost-redux/selectors/entities/posts';
+import {makeGetDraftsByPrefix, DraftSelector} from 'selectors/drafts';
+import {GlobalState} from 'types/store';
+import {StoragePrefixes, StorageTypes} from 'utils/constants';
+
+const getChannelDrafts: DraftSelector = makeGetDraftsByPrefix(StoragePrefixes.DRAFT);
+const getThreadDrafts: DraftSelector = makeGetDraftsByPrefix(StoragePrefixes.COMMENT_DRAFT);
+
+// Drafts pre global drafts are missing the channelId in the value
+// thus making it hard to determine in which team they belong.
```

Comment:
Is this needed to filter the drafts by team (not sure if we want that, since, for example, for searches, saved posts, etc... we have started to move towards showing in all teams) or to get the right link to jump to?

I know that for posts, there is a way to jump to the post without specifying the team. For channels I am not sure.

Reply:
Yep considering that I added `channelId`. It could work by adding team id, but the thing is that when we move the drafts from local storage to the DB we are going to store the channel id, so I thought this would make it easier.

Reply:
@esethna Do you have any thoughts on whether the draft view will eventually be universal and not team scoped?